orangecontrib/text/tests/test_preprocess.py

```diff
@@ -73,23 +92,91 @@ def test_porter_stemmer(self):
+        stemmed = ['caress', 'fli', 'die', 'mule', 'deni', 'die', 'agre', 'own', 'humbl', 'size', '.']
+
+        p = Preprocessor(lowercase=True, stop_words=None,
+                         transformation=PorterStemmer, use_twitter_tokenizer=False)
+        corpus = p(corpus)[0]
+        self.assertEqual(corpus, stemmed)
+
+    # Snowball stemmer.
+    def test_snowball_stemmer(self):
+        words = ['caresses', 'flies', 'dies', 'mules', 'denied',
+                 'died', 'agreed', 'owned', 'humbled', 'sized',
+                 'meeting', 'stating', 'siezing', 'itemization',
+                 'sensational', 'traditional', 'reference', 'colonizer',
+                 'plotted']
+        stems = ['caress', 'fli', 'die', 'mule', 'deni', 'die', 'agre', 'own',
+                 'humbl', 'size', 'meet', 'state', 'siez', 'item', 'sensat',
+                 'tradit', 'refer', 'colon', 'plot']
 
-    # TODO: uncomment this test when the preprocessing  will be implemented
-    #def test_porter_sentence(self):
-    #    corpus = ['Caresses flies dies mules denied died agreed owned humbled sized.']
-    #    stemmed = ['caress', 'fli', 'die', 'mule', 'deni', 'die', 'agre', 'own', 'humbl']
+        for w, s in zip(SnowballStemmer(words), stems):
+            self.assertEqual(w, s)
 
-    #    p = Preprocessor(lowercase=True, stop_words=None, trans=Stemmer)
-    #    corpus = p(corpus)
-    #    self.assertEqual(sorted(corpus), sorted(stemmed))
+    def test_snowball_sentence(self):
+        corpus = ['Caresses flies dies mules denied died agreed owned humbled sized.']
+        stemmed = ['caress', 'fli', 'die', 'mule', 'deni', 'die', 'agre', 'own', 'humbl', 'size', '.']
 
+        p = Preprocessor(lowercase=True, stop_words=None,
+                         transformation=SnowballStemmer, use_twitter_tokenizer=False)
+        corpus = p(corpus)[0]
+        self.assertEqual(corpus, stemmed)
 
-    def test_wordnet_lematizer(self):
+    # Lemmatizer.
+    def test_wordnet_lemmatizer(self):
         words = ['dogs', 'churches', 'aardwolves', 'abaci', 'hardrock']
         lemas = ['dog', 'church', 'aardwolf', 'abacus', 'hardrock']
 
-        for w, s in zip(words, lemas):
-            self.assertEqual(Lemmatizer(w), s)
+        for w, s in zip(Lemmatizer(words), lemas):
+            self.assertEqual(w, s)
+
+    def test_wordnet_lemmatizer_sentence(self):
+        corpus = ['Pursued brightness insightful blessed lies held timelessly minds.']
+        lemmas = ['pursued', 'brightness', 'insightful', 'blessed', 'lie', 'held', 'timelessly', 'mind', '.']
 
+        p = Preprocessor(lowercase=True, stop_words=None,
+                         transformation=Lemmatizer, use_twitter_tokenizer=False)
+        corpus = p(corpus)[0]
+        self.assertEqual(corpus, lemmas)
+
+    def test_full_corpus_preprocess(self):
+        corpus = Corpus.from_file(os.path.join(get_sample_corpora_dir(), 'deerwester.tab'))
+        p = Preprocessor(lowercase=True, stop_words=None)
```

Comment:
I would remove the passing of default parameter values, such as `lowercase=True, stop_words=None`.